WootricSDK/WootricSDK/WTRSlider.m

```diff
@@ -63,7 +63,8 @@ - (instancetype)initWithSuperview:(UIView *)superview viewController:(UIViewCont
     [self setThumbImage:image forState:UIControlStateHighlighted];
     [self setTranslatesAutoresizingMaskIntoConstraints:NO];
     [self addTarget:superview action:NSSelectorFromString(@"updateSliderScore:") forControlEvents:UIControlEventValueChanged];
-    UITapGestureRecognizer *gr = [[UITapGestureRecognizer alloc] initWithTarget:viewController action:NSSelectorFromString(@"sliderTapped:")];
+    UILongPressGestureRecognizer *gr = [[UILongPressGestureRecognizer alloc] initWithTarget:viewController action:NSSelectorFromString(@"sliderTapped:")];
```

Comment:
So was it working with tap before but not with long press? Now it will work with long press but not tap ?

Reply:
@prabhatjha that's why I'm setting the `minimumPressDuration` to 0 in the next line. So no matter if it's a tap or a long press, the `sliderTapped` method would be called.

Reply:
Makes sense!